PySide/config.py

```diff
@@ -43,6 +43,12 @@
 
 		},
 
+		"symbolicLinks" : [
+
+			( "{buildDir}/bin/pyside2-uic", "../lib/Python.framework/Versions/Current/bin/pyside2-uic" ),
```

Comment:
Is there any merit in just putting `lib/Python.framework/Versions/Current/bin` on `$PATH` in the gaffer wrapper? Rather than relying on making a bunch of symlinks?

Reply:
Possibly, but that would put all sorts of things on the path that I don't think we ship for the linux builds (idle, pydoc, shiboken2, smptd.py etc). It also doesn't solve the problem I'm solving here, which all has to happen before the gaffer wrapper is a thing,

Reply:
Ahh yeah. Fair enough. Should we also include the other `pyside2-*` utils too?

Reply:
Possibly, but I'm trying not to think about what we ship right now. Right now I'm trying to make the minimal fix so that we can actually _build_, while leaving what we ship unchanged.

Reply:
Noting the conclusion from our off-line chat, (which as ever, was on-line :upside_down_face:). There may well be a more holistic version of this, but there are other :fish: to :shallow_pan_of_food: right now.